ui/components/component-library/button-link/button-link.test.js

```diff
@@ -81,7 +81,7 @@ describe('ButtonLink', () => {
   });
   it('should render with icon', () => {
     const { container } = render(
-      <ButtonLink data-testid="icon" iconName="add-square" />,
+      <ButtonLink data-testid="icon" startIconName="add-square" />,
```

Comment:
Should be able to use `ICON_NAMES` object here. Will help when we add and remove icons